### ISSUE TITLE
originswag.eth.link and other dweb stores break on localhost

### ISSUE THREAD

**Issue by lidel** [ipfs-shipyard/ipfs-companion#872]
I was able to reproduce the issue reported by @andyschwab, 
it is present in both Firefox and Chromium.

- Opening https://originswag.eth.link with Companion enabled ends up on https://originswag.eth.link.ipns.localhost:8080/#/
- http://originswag.eth.link.ipns.localhost:8080/ is opened, subresources are fetched but then suddenly URL gets replaced with HTTPS one: https://originswag.eth.link.ipns.localhost:8080/
- this is because `http://originswag.eth.link.ipns.localhost:8080/dist/vendors~app.0f591475.bundle.js` has this snippet:
  `window.location.href=window.location.href.replace("http:","https:")` 
- I believe `http:` should  be allowed, when store is loaded from `http://*.localhost`

**tl;dr** store software needs a fix to allow loading from local IPFS gateway at  `http://*.localhost:*/`

**Comment by andyschwab** [comment]
Reached out to Origin to alert them to the issue.
